concourse/pipelines/guest-package-build.yaml

```diff
@@ -149,40 +149,60 @@ jobs:
+      - task: build-package-image-debian-11-arm64
+        file: guest-test-infra/concourse/tasks/daisy-build-package-image.yaml
+        vars:
+          source-image: projects/debian-cloud-testing/global/images/family/debian-11-arm64
+          dest-image: debian-11-arm64-((.:build-id))
+          gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent_((.:package-version))-g1_arm64.deb
+          machine-type: t2a-standard-2
+          worker-image: projects/compute-image-tools/global/images/family/debian-11-worker-arm64
       - task: build-package-image-centos-7
         file: guest-test-infra/concourse/tasks/daisy-build-package-image.yaml
         vars:
           source-image: projects/centos-cloud/global/images/family/centos-7
           dest-image: centos-7-((.:build-id))
           gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent-((.:package-version))-g1.el7.x86_64.rpm
+          machine-type: e2-medium
+          worker-image: projects/compute-image-tools/global/images/family/debian-10-worker
       - task: build-package-image-rhel-7
         file: guest-test-infra/concourse/tasks/daisy-build-package-image.yaml
         vars:
           source-image: projects/rhel-cloud/global/images/family/rhel-7
           dest-image: rhel-7-((.:build-id))
           gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent-((.:package-version))-g1.el7.x86_64.rpm
+          machine-type: e2-medium
+          worker-image: projects/compute-image-tools/global/images/family/debian-10-worker
       - task: build-package-image-rhel-8
         file: guest-test-infra/concourse/tasks/daisy-build-package-image.yaml
         vars:
           source-image: projects/rhel-cloud/global/images/family/rhel-8
           dest-image: rhel-8-((.:build-id))
           gcs-package-path: gs://gcp-guest-package-uploads/guest-agent/google-guest-agent-((.:package-version))-g1.el8.x86_64.rpm
+          machine-type: e2-medium
+          worker-image: projects/compute-image-tools/global/images/family/debian-10-worker
   - task: guest-agent-image-tests
     file: guest-test-infra/concourse/tasks/image-test-args.yaml
     vars:
-      images: projects/gcp-guest/global/images/debian-9-((.:build-id)),projects/gcp-guest/global/images/debian-10-((.:build-id)),projects/gcp-guest/global/images/debian-11-((.:build-id)),projects/gcp-guest/global/images/centos-7-((.:build-id)),projects/gcp-guest/global/images/rhel-7-((.:build-id)),projects/gcp-guest/global/images/rhel-8-((.:build-id))
+      images: projects/gcp-guest/global/images/debian-9-((.:build-id)),projects/gcp-guest/global/images/debian-10-((.:build-id)),projects/gcp-guest/global/images/debian-11-((.:build-id)),projects/gcp-guest/global/images/debian-11-arm64-((.:build-id)),projects/gcp-guest/global/images/centos-7-((.:build-id)),projects/gcp-guest/global/images/rhel-7-((.:build-id)),projects/gcp-guest/global/images/rhel-8-((.:build-id))
```

Comment:
i don't think this will work - the VMs are launched by default on e2 family instances. we can't intermix x86 and arm64 images in one test run unless we make some changes to CIT. so probably we just want to add a second task (can be in parallel with this one) that specifies the t2a machine type, the arm-enabled zone, and the arm image.

Reply:
My understanding is that, the "manager"  is just a launcher for the test workflows. So, ideally, it can launch whatever machine type dynamically depending on the test image. I thought that has been done.

Let me put it in a separate task for now.

Reply:
the 'manager' is an internal implementation detail of CIT; CIT will automatically detect the need to specify ARM binaries if you're using ARM machine type, but you can't launch all of these images on the same machine type, and the machine type is not automatically detected but passed in as an argument

Reply:
done